content/kubermatic/main/tutorials-howtos/applications/add-applications-to-cluster/_index.en.md

```diff
@@ -1,13 +1,11 @@
 +++
 title = "Adding Applications To A Cluster"
 date =  2022-08-03T16:43:41+02:00
-weight = 5
+weight = 2
 +++
-## Introduction
 
 This guide targets Cluster Admins and details how KKP Applications can be integrated into a cluster.
```

Comment:
Since this section covers a lot of different ways to install, I would recommend to add a Table of Contents to it. If I remember correctly this was changed recently and now toc is a page parameter

```
+++
..
enableToc = true
+++
```

Reply:
Heads up, TOC does not need to be set anymore because the docs now feature a sidebar TOC on the right side. I'd recommend to rebase the branch if this doesn't show up in previews yet.

Reply:
yes TOC is included on all pages by default on the right in the `On this page` section or "information" icon if you have folded the section.